### PR TITLE
1375 update chromebook question and validation

### PR DESCRIPTION
**Pull request by aldavidson**
### Context

[Trello card 1375](https://trello.com/c/oaDQbQTn/1375-update-chromebook-question-and-validation) -  In user research, we saw users getting confused by what the G Suite for Education domain was. Improve the labelling to make it clearer.

### Changes proposed in this pull request

* label the domain field appropriately using the `institution_type` and `responsible_body.humanized_type` where appropriate 
* new `chromebook_domain_label` method on `ChromebookInformationForm` to encapsulate the logic
* improve wording on error message when the domain is invalid

### Guidance to review

(see the Trello card and [Slack thread](https://ukgovernmentdfe.slack.com/archives/C01A7DVKE9J/p1611761686137200) for discussion)

When the school's responsible body is a Local Authority:
Screenshot from 2021-01-28 10-36-29

When it's a Further Education school:
Screenshot from 2021-01-28 10-39-37

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -456,9 +456,9 @@ en:
             will_need_chromebooks:
               blank: Tell us whether the school will need Chromebooks
             school_or_rb_domain:
-              blank: Enter a domain in the right format, like myschool.org.uk
-              invalid: Enter a domain in the right format, like myschool.org.uk
-              invalid_domain: Enter a domain registered for G Suite for Education
+              blank: Enter an email domain registered for G Suite for Education
+              invalid: Enter an email domain registered for G Suite for Education
```
Comment (fofr) [inline config/locales/en.yml:460]: I don't like that we're removing the examples here. Why are we not keeping the "in the right format" message?

Comment (aldavidson) [reply config/locales/en.yml:460]: Question for @ralph-hawkins ?

Comment (aldavidson) [reply config/locales/en.yml:460]: I've added the ', like myschool.org.uk' back in to the error messages, what do you think now?

Comment (ralph-hawkins) [reply config/locales/en.yml]: Yeah — sorry that wasn't my intention to change all the error messages — they're better kept specific.

File: config/locales/en.yml [hunk 2]
```diff
@@ -456,9 +456,9 @@ en:
             will_need_chromebooks:
               blank: Tell us whether the school will need Chromebooks
```
Comment (ralph-hawkins) [inline config/locales/en.yml:457]: Can we change "school" to school.institution_type?

File: app/helpers/view_helper.rb [hunk 3]
```diff
@@ -176,6 +176,13 @@ def link_to_ed_settings_form
     govuk_link_to 'DfE educational settings status form', 'https://form.education.gov.uk/service/educational-setting-status'
   end
 
+  def chromebook_domain_label(school)
+    label = Array(school.institution_type.capitalize)
+    label << "or #{school.responsible_body.humanized_type}" unless school.is_a?(FurtherEducationSchool)
```
Comment (jamesrose) [inline app/helpers/view_helper.rb:181]:
```suggestion
    label << "or #{school.responsible_body.humanized_type}" unless school.is_further_education?
```

**Review by jamesrose — APPROVED** [review]
One minor thing, but looks good to me. I'll defer again to @ralph-hawkins or @fofr on the rest.

File: config/locales/en.yml (hunk 2 above)
Comment (aldavidson) [reply config/locales/en.yml]: I've got this working in 7bc68bbe, and refactored `spec/form_objects/chromebook_information_form_spec.rb` to test the new behaviour - can you take another look?

**Review by ralph-hawkins — APPROVED** [review]
👍  looks good to me

**Pull request merged**